packages/tables/stubs/Table.stub

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace {{ namespace }};
+
+use App\Models\{{ model }};
+use Filament\Tables;
+use Filament\Tables\Concerns\InteractsWithTable;
+use Filament\Tables\Contracts\HasTable;
+use Livewire\Component;
+use Illuminate\Contracts\View\View;
+use Illuminate\Database\Eloquent\Builder;
+
+class {{ class }} extends Component implements HasTable
+{
+    use InteractsWithTable;
+
+    protected function getTableQuery(): Builder
+    {
+        return {{ modelClass }}::query();
+    }
+
+    protected function getTableColumns(): array
+    {
+        return [
+{{ columns }}
```

Comment:
Is this indentation on purpose?

Reply:
Yes, because we use `$this->indentString()` to indent potentially multiple lines of columns